pom.xml

```diff
@@ -201,20 +201,20 @@
                                 <!-- Services used in test suite -->
                                 <keycloak.image>quay.io/keycloak/keycloak:16.1.0</keycloak.image>
                                 <rhsso.73.image>registry.access.redhat.com/redhat-sso-7/sso73-openshift</rhsso.73.image>
-                                <postgresql.13.image>quay.io/bitnami/postgresql:13.5.0</postgresql.13.image>
-                                <elastic.71.image>quay.io/bitnami/elasticsearch:7.10.2</elastic.71.image>
-                                <mysql.57.image>quay.io/bitnami/mysql:5.7.32</mysql.57.image>
-                                <mysql.bitnami.80.image>quay.io/bitnami/mysql:8.0</mysql.bitnami.80.image>
+                                <postgresql.13.image>postgres:13.6</postgresql.13.image>
+                                <elastic.71.image>elasticsearch:7.17.2</elastic.71.image>
+                                <mysql.57.image>mysql:5.7.36</mysql.57.image>
+                                <mysql.upstream.80.image>mysql:8.0</mysql.upstream.80.image>
```

Comment:
I am not particularly happy with adding `upstream` image just to single upstream image while keeping the rest of upstream ones umarked - this is misleading naming.

We use fewer authenticated container images - perhaps we should tag all of these somehow instead, WDYT?

Reply:
It would be a nice idea, but I would prefer to discuss and implement it some other time, when we do not have all our tests failing without a fix. As a temporary solution I can rename this property  into `mysql.8.image`

Reply:
Kindly create an issue and add a TODO with reference.